common/templates/mathjax_include.html

```diff
@@ -85,6 +94,21 @@
             explorer: true
         }
     };
+    window.addEventListener('resize', MJrerender);
```

Comment:
The listener should be for both `mathjax_mode = 'wiki'` as well.

Reply:
I don't understand what this is asking for

Reply:
nm, I see.